ci-operator/config/openshift/hive/openshift-hive-master.yaml

```
@@ -174,6 +174,58 @@ tests:
         requests:
           cpu: 100m
     workflow: generic-claim
+- as: e2e-assisted
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-1
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.11"
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

```suggestion
  optional: true
  run_if_changed: clusterinstall
```

`optional: true` is so this only runs when requested (or when a path containing `clusterinstall` changes, per above).